### DISCUSSION TITLE
Reusing exisitng db (migrate=False)

### DISCUSSION THREAD

**Discussion by KellerKev** [emmett-framework/emmett#329] [Q&A] [answered]
What is the best practice to reuse an exisitg db with tables in it?

I am used to define the fields in pydal and set migrate=False.

Is it similiar in Emmett? 

Thank you!

**Comment by gi0baro** [comment, marked as answer]
Under default behaviour Emmett's ORM has automatic schema migrations disabled. You can safely connect to any existing database. See https://emmett.sh/docs/2.2.x/orm/migrations

**Comment by KellerKev** [comment]
Awesome! Sorry I did not read this yet. I had a read now. I love the approach. Just one last thing I could not find. How do I enable automatic migration if I wanted to? Thanks !

**Reply by gi0baro** [reply]
with `auto_migrate=True` parameter on the `Database` class. But _I strongly discourage its usage_ 😉
